api/bases/glance.openstack.org_glances.yaml

```diff
@@ -120,6 +120,8 @@ spec:
                   in /etc/<service> . TODO: -> implement'
                 type: object
               glanceAPIExternal:
+                default:
+                  replicas: 1
```

Comment:
Thanks, this looks good to me, and it's actually a good idea providing this level of flexibility.